### ISSUE TITLE
Support for Eslint v9

### ISSUE THREAD

**Issue by hiteto218** [brendanmorrell/eslint-plugin-styled-components-a11y#62]
Hello,
It seems that Eslint v9 has been newly released. 
However, when I execute it with v9, I encountered the following error:
```
Error: Key "rules": Key "styled-components-a11y/interactive-supports-focus":
Value [{"tabbable":["button","checkbox","link","searchbox","spinbutton","switch","textbox"]}] should NOT have more than 0 items.
```
Do you have any plans to support v9?

**Comment by brendanmorrell** [comment]
Was unaware of the error (or an eslint version update for that matter), but
i certainly plan on supporting newer versions as long as the plugin remains
useful. Will probably be a couple weeks until i have time to take a look,
but ill keep you posted. And if you have time to take a look, Id be happy
to look at a fix PR

**Comment by brendanmorrell** [comment]
so, finally getting around to looking at this. it looks like v9 of eslint involves pretty sweeping changes, to the point that most plugins do not currently support it. from what i see, eslint-plugin-jsx-a11y falls into that category, and [does not yet support v9](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/978).  this plug in is built on top of eslint-plugin-jsx-a11y, and as such, I don't think there is any way for me to support v9 at this point. if i am wrong about that, let me know and i can reopen this. if so, it would also help if you could provide a code sandbox or a repo using eslint v9 which showcases a component correctly getting a jsx-a11y error (such as an onClick on a div showing the 'jsx-a11y/click-events-have-key-events' error so i can verify everything is working with that, and then i can add eslint-plugin-styled-components-a11y and work backward from there until a styled component produces the same error)

**Comment by brendanmorrell** [comment]
fixed in version [2.1.35](https://www.npmjs.com/package/eslint-plugin-styled-components-a11y/v/2.1.35)
